doc-Migration_Analytics_Guide/cfme/modules/Migration-analytics-report.adoc

```diff
@@ -13,15 +13,18 @@ This section describes the estimated savings realized by migrating from VMware t
 
 .Workload migration summary
 
-This section describes the complexity of the migration and provides detailed workload data. A pie chart displays the complexity of migrations:
+This section describes the complexity of the migration and provides detailed workload data.
 
-* *Easy*: Virtual machine with no unusual configuration
-* *Medium*: For example, virtual machine with more than four vNICs or a shared disk
-* *Hard*: For example, a virtual machine with Raw Device Mapping
-* *Unsupported*: Operating system is not supported for migration, for example, FreeBSD 12
+A pie chart summarizes the complexity of the migration:
+
+* *Easy*: Virtual machine has a supported operating system and configuration. It requires no changes for migration.
+* *Medium*: Virtual machine configuration, for example, Windows operating system, high availability configuration, cluster membership, or shared disks, may require manual changes after migration.
+* *Hard*: Virtual machine configuration, for example, raw device mapping, requires manual changes before and after migration.
+* *Unknown*: Virtual machine configuration is incomplete. Its migration complexity cannot be determined.
+* *Unsupported*: Virtual machine configuration, for example, an unsupported operating system, is not supported on the target platform.
```

Comment:
Here we can give an example for unsupported os like Ubuntu, Debian etc

Reply:
Good idea. We had Debian before. Can add